WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -370,23 +370,6 @@ extension PostEditor where Self: UIViewController {
         self.postEditorStateContext.updated(isBeingPublished: false)
     }
 
-    /// Uploads the post
-    ///
-    /// - Parameters:
-    ///     - completion: the closure to execute when the publish operation completes.
-    ///
-    private func uploadPost(completion: ((_ post: AbstractPost?, _ error: Error?) -> Void)?) {
-        mapUIContentToPostAndSave(immediate: true)
```

Comment:
This line seems important to retain. Are we sure about not keeping it?

Reply:
Nice catch!

Done in 1303bbe4cc7c646364d795f3377c5a15f8ebeabd